Formula/deno.rb

```diff
@@ -0,0 +1,17 @@
+class Deno < Formula
+  desc      "Secure TypeScript runtime on V8"
+  homepage  "https://github.com/denoland/deno/"
+  url       "https://github.com/denoland/deno/releases/download/v0.2.4/deno_osx_x64.gz"
```

Comment:
For formula to be accepted in the core tap, it should be built from source.

Please refer to the [Acceptable Formulae](https://docs.brew.sh/Acceptable-Formulae) section from our documentation.